minio/minio
# Update links to documentation site

## Description

The relaunch of the documentation website has caused all pages to have new URLs.

This PR updates all `docs.min.io` and `docs.minio.io` links to point to the most similar page at `min.io/docs/minio/*`.

## Motivation and Context

New URLs for the docs site.

## How to test this PR?

The new docs have been platformized.
There are separate sets of docs for Windows, Linux, MacOS, Containers, and Kubernetes/Upstream. (With a few other Kubernetes vendors to come.)

Most links point to the Linux set of docs.
Docker and Docker Compose should point to the Container set of docs.
Kubernetes specific situation should point to the Kuberenetes/upstream set of docs.

(The set of docs is reflected in the URL of the doc page.)

## Types of changes

## Checklist:
